dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22ComponentTesterTests.java

```diff
@@ -51,6 +52,7 @@
  * Tests to execute on the jsfTestServer2 that use HtmlUnit.
  */
 @Mode(TestMode.FULL)
+@CheckpointTest(alwaysRun = true)
```

Comment:
I think this is not necessary?  Is this a checkpoint tests?

Reply:
I completely overlooked this. Thanks for spotting it.